packages/rich-text/src/helpers/editor.ts

```diff
@@ -285,3 +294,26 @@ export function currentSelectionPrecedesTableCell(editor: PlateEditor): boolean
     !!nextNode && TABLE_BLOCKS.includes(nextNode.type as BLOCKS) && isAtEndOfTextSelection(editor)
   );
 }
+
+/**
+ * It filters out all paragraphs and headings from a node entry (node + path) based on its path and convert them into paragraphs.
+ */
+export function slateNodeEntryToText(
+  editor: PlateEditor,
+  nodeEntry: SlateNodeEntry
```

Comment:
Why require an entire `NodeEntry` here if all we use is its `path`?

Reply:
That's what my question on the description is about.